docs/date.md

```diff
@@ -42,4 +44,23 @@ for common years, and 1-366 for leap years.
 isLeapYear(year)
 ```
 
-`isLeapYear` returns true if the given year is a leap year.
\ No newline at end of file
+`isLeapYear` returns true if the given year is a leap year.
+
+### fn parseRFC3339
+
+```ts
+parseRFC3339(input)
+```
+
+`parseRFC3339` parses an RFC3339-formatted date & time string into an object containing the 'year', 'month', 'day', 'hour', 'minute' and 'second fields.
```

Comment:
Would be useful to have an example of an RFC3339 date/time for those of us that know the format but not the RFC number.

Reply:
Sure, followed up in https://github.com/jsonnet-libs/xtd/pull/24